flareon-macros/tests/ui/attr_main_args.stderr

```diff
@@ -0,0 +1,11 @@
+error: flareon::main function must have zero arguments
+ --> tests/ui/attr_main_args.rs:2:15
+  |
+2 | async fn main(arg: i32) -> flareon::Result<FlareonProject> {
+  |               ^^^^^^^^
+
+error[E0601]: `main` function not found in crate `$CRATE`
```

Comment:
Is this supposed to be `$CRATE` or is that some mistake?

Reply:
Yes, the `trybuild` crate [normalizes error messages](https://github.com/dtolnay/trybuild/blob/96340263d9eb0705f98f0fa18b1281d2e9306347/src/normalize.rs#L440).